etc/containerpilot.json

```diff
@@ -24,12 +24,13 @@
     {
       "name": "snapshot_check",
       "command": "python /usr/local/bin/manage.py snapshot_task",
-      "frequency": "10s",
```

Comment:
At the end of the first pass thru the health check for the primary we do an initial snapshot so we can bootstrap replication (at the end of `run_as_primary`). Before we moved the snapshot into its own task we also checked if we needed a snapshot at the end of the health check. This worked fine because we'd already completed the `run_as_primary` steps. But when we moved it into its own task it overlaps with the health check, which means it can start to run before we've completed `run_as_primary` and this creates a ton of logging noise and errors. By moving it to 5 min we know that the initial setup has been completed without having to recheck it every time.